packages/foreman/pcp-mmvstatsd/pcp-mmvstatsd.spec

```diff
@@ -42,13 +41,6 @@ install -D -m 755 _gopath/src/%{name}-%{version}/%{name}-%{version} %{buildroot}
 install -D -m 644 %{SOURCE1} %{buildroot}%{_unitdir}/%{name}.service
 install -D -m 644 %{SOURCE2} %{buildroot}%{_sysconfdir}/default/%{name}
 
-%pre
-getent group %{name} >/dev/null || groupadd -r %{name}
-getent passwd %{name} >/dev/null || \
-    useradd -r -g %{name} -d %{_sharedstatedir}/%{name} -s /sbin/nologin \
-    -c "Prometheus services" %{name}
-exit 0
-
```

Comment:
Removed the user, the daemon will run under `pcp` as it's the only option for shared memory. Since this was an initial RPM not distributed yet in any release, I don't do any deletion of existing user.